spec/factories/state_file_md_intakes.rb

```diff
@@ -130,8 +130,6 @@
     primary_middle_initial { "A" }
     primary_last_name { "Lando" }
     primary_birth_date { Date.new(1950, 01, 01) } # matches the bday in md_minimal.json
-    primary_signature_pin { '12345' }
-    primary_esigned_at { DateTime.now }
```

Comment:
moved setting of the signature-related attributes in the actual specs that need them, instead of treating them as a base-line state. We'd ideally would like to have intakes that start with no signatures, since for the most of our in-app flow the app is not signed